config/settings.yml

```diff
@@ -93,6 +93,7 @@ discourse_ai:
   ai_openai_gpt4_url: "https://api.openai.com/v1/chat/completions"
   ai_openai_gpt4_32k_url: "https://api.openai.com/v1/chat/completions"
   ai_openai_embeddings_url: "https://api.openai.com/v1/embeddings"
+  ai_openai_organization:
```

Comment:
All site settings require a default value to be set. I think we have some checks for this in core but apparently not for plugins.

Reply:
weird... seems though like pointless ceremony?

Reply:
it return null now that seems to be correct according to code ... anyway ... will give all the blank ones an empty string...